.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# Code owners for renovate-config
+# https://help.github.com/en/articles/about-code-owners
+
+* @literat
```

Comment:
I tend to prefer to use our corporate emails. In that case you transparently see the owner, without the additional layer of GH usernames... Not everyone has obvious username. Also, once your email is de-assigned from your account, you will stop being codeowner.

Reply:
Ok, but as I understand this correctly, this is the feature of the enterprise account only...

> In most cases, you can also refer to a user by an email address that has been added to > their account, for example user@example.com. You cannot use an email address to refer > to a managed user account. For more information about managed user accounts, see [About Enterprise Managed Users](https://docs.github.com/en/enterprise-cloud@latest/admin/identity-and-access-management/using-enterprise-managed-users-for-iam/about-enterprise-managed-users)" in the GitHub Enterprise Cloud documentation.

Or am I wrong? Anyway, thanks for enlightening me.

Reply:
I think it should work, the don't use Enterprise Managed Users, IMO? At least it works for me :-D 

But you must have added the email in your settings.

Reply:
Fixed. There was @almamedia one, not the `career` in my settings. Now it seems valid.